ext/sodium/libsodium.c

```diff
@@ -2794,6 +2808,73 @@ PHP_FUNCTION(sodium_compare)
 }
 #endif
 
+#ifdef HAVE_AESGCM
+PHP_FUNCTION(sodium_crypto_aead_aes256gcm_keygen)
```

Comment:
All these new functions should use zend_parse_parameters_none() for consistency across php-src